### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'misc/sitter-libs'

### ISSUE THREAD

**Issue by Roreyou** [Intelligent-CAT-Lab/AlphaTrans#1]
Hello! I'm encountering an issue when I try to run the `bash scripts/decompose_test.sh` following the instructions in the README. The error I received seems to be related to a missing directory (misc/sitter-libs), but I cannot find this directory anywhere in the project. I would really appreciate it if you could provide any help in resolving this error or any explanations to clarify my misunderstanding. Thanks!

**Comment by FreitasVarejo** [comment]
I ahve also ran into thius problem

**Comment by mrigankpawagi** [comment]
Hi @Roreyou! Could you please share the complete error log? That would better help us understand the error.

**Comment by alibrahimzada** [comment]
@FreitasVarejo @Roreyou thank you for your interest in our work. We are aware of some missing information in our work. To handle all issues, we are working on containerizing AlphaTrans. We should be releasing it next week.

**Comment by Roreyou** [comment]
> Hi @Roreyou! Could you please share the complete error log? That would better help us understand the error.

Of course! Here is the the complete error log after I run `bash scripts/decompose_test.sh`.
![image](https://github.com/user-attachments/assets/dce12916-d491-4db2-9040-5a473ac1da12)

**Comment by alibrahimzada** [comment]
@Roreyou can you please try again with respect to the new README? We containerized AlphaTrans. Please let us know if you face anything else by replying here or opening a new issue.

**Comment by Roreyou** [comment]
Hi! I encoutner the following error when I run `bash scripts/decompose_test.sh`. 
![image](https://github.com/user-attachments/assets/3132fbd0-e9ba-4e43-9727-47ba93897363)

Then I found a similar directory named `source_test_execution_decomposed_tests` under the `data` folder. 
![image](https://github.com/user-attachments/assets/e0de15d0-0607-4f07-b233-c6155a8e2e76)

I guess I need to generate the similar files(`test.json` and `coverage.json`) under a folder named  `source_test_execution` folder  before I do the test decomposition? But I have no idea how to do it.

**Comment by mrigankpawagi** [comment]
Thanks for bringing this up, @Roreyou. We are working on fixing the instructions in README and would get back to you soon.

**Comment by alibrahimzada** [comment]
@Roreyou we improved our README file and moved most of the work in Dockerfile. Can you please try again based on the new README and let us know if you have more issues? We appreciate your time testing our tool.
